pkg/validations/createvalidations/createvalidations.go

```diff
@@ -4,6 +4,16 @@ import (
 	"github.com/aws/eks-anywhere/pkg/validations"
 )
 
+// string values of supported validation names that can be skipped.
+const (
+	VSphereUserPriv = "vsphere-user-privilege"
```

Comment:
We're going to document a list of these right? is there any way we can automate publishing what these are?

Reply:
Publishing on the docs you mean? 
I dont think so, new flags we add will get listed with `--help` command but not sure if we can send that to the docs.